main.py

```diff
@@ -42,13 +46,17 @@
     # ----------------------------------------
     #               MARKET
     # ----------------------------------------
-    if message.content.startswith('!m'):
+    elif message.content.startswith('!m'):
```

Comment:
Whilst this is fine, it is slightly inefficient as you `return` out of the above if statement anyway, so there isn't any point wasting processing time checking if it failed or not

Reply:
oh, didnt know this is the way in Python, in C#, this woud be more efficent, and